windows/src/developer/setup/manifest.in

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <assembly manifestVersion="1.0" xmlns="urn:schemas-microsoft-com:asm.v1">
-	<assemblyIdentity type="win32" name="tavultesoft.keymandeveloper.setup" version="$VERSION" processorArchitecture="x86"/>
+	<assemblyIdentity type="win32" name="tavultesoft.keymandeveloper.setup" version="$$VersionWin" processorArchitecture="x86"/>
```

Comment:
Does windows/src/developer/TIKE/manifest.in also need this change?

Reply:
also noticed there's an extra `$`

Reply:
Wow, only just spotted this booboo -- `$$VersionWin` instead of `$VersionWin`. And yes, I should probably correct the other manifest.in.

Reply:
windows/src/developer/TIKE/manifest.in already has the change?

Reply:
oh right - nvm